README.md

```diff
@@ -1,2 +1,67 @@
 # act-bench
-Activation function benchmark datasets Act-Bench-CNN, Act-Bench-ResNet, and Act-Bench-ViT
+Activation function benchmark datasets `Act-Bench-CNN`, `Act-Bench-ResNet`, and `Act-Bench-ViT`
+
+---
+
+## Getting Started
+
+Clone the repository and download the datasets.
+
+```bash
+git clone https://github.com/cognizant-ai-labs/act-bench.git
+cd act-bench
+bash download-datasets.sh
+```
+
+In total, the datasets are about 408 MB in size.
+
+---
+
+## Loading the Datasets
+
+To load the datasets, unpickle them using the `dill` module.
+
+```python
+import dill as pickle
+
+with open('datasets/act-bench-cnn.pkl', 'rb') as f:
+    act_bench_cnn = pickle.load(f)
+
+with open('datasets/act-bench-resnet.pkl', 'rb') as f:
+    act_bench_resnet = pickle.load(f)
+
+with open('datasets/act-bench-vit.pkl', 'rb') as f:
+    act_bench_vit = pickle.load(f)
```

Comment:
Pickle is the devil.  Especially so when it comes to long-term storage.
The format itself is not human-readable for the (rightly) security-sheepish.
Underlying data structures change over time posing maintenance nightmares and occasional security breaches depending on what is actually stored (think eval()-ed code stored as strings).

Please strongly consider changing your data format.
Are there any other standard formats that will do the job? JSON? CSV?

Reply:
Okay, will finagle with JSON

Reply:
Okay, done!